django/db/migrations/loader.py

```diff
@@ -186,15 +186,20 @@ def build_graph(self):
         # This is just for dependency re-pointing when applying replacements,
         # so we ignore run_before here.
         reverse_dependencies = {}
-        for key, migration in normal.items():
+        reverse_dependencies_normal = {}
```

Comment:
I'm not too familiar with this code but the "_normal" suffix doesn't make the variable name self-explanatory to me.